mmcv/arraymisc/quantization.py

```diff
@@ -29,13 +35,17 @@ def quantize(arr, min_val, max_val, levels, dtype=np.int64):
     return quantized_arr
 
 
-def dequantize(arr, min_val, max_val, levels, dtype=np.float64):
+def dequantize(arr: np.ndarray,
+               min_val: Union[int, float],
+               max_val: Union[int, float],
+               levels: int,
+               dtype: np.type = np.float64) -> tuple:
```

Comment:
'numpy' has no attribute 'type'

Reply:
So what is the `np.float64` type? When I try  `type(np.int64)` it returns `<class 'type'>`, so is this type hint `type` ?

Reply:
np.float64 is a class and its type is metaclass `type` so we can ignore the type hint for `dtype`.